docs/reference/generated/alert-dialog-root.json

```diff
@@ -12,16 +12,19 @@
       "description": "Whether the dialog is currently open."
     },
     "onOpenChange": {
-      "type": "(open, event, reason) => void",
+      "type": "((open: boolean, event: Event | undefined, reason: \"click\" | \"hover\" | \"focus\" | \"focus-out\" | \"escape-key\" | \"outside-press\" | \"trigger-press\" | undefined) => void)",
       "description": "Event handler called when the dialog is opened or closed."
     },
     "actionsRef": {
-      "type": "{ current: { unmount: func } }",
+      "type": "RefObject",
```

Comment:
Was this overriden before? Maybe we can improve the type here?

Reply:
Refs are pretty hard to handle because of a certain TS weirdness. I'll try to make it more readable somehow, though.